<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlacshEffectAttaqueDial</class>
 <widget class="QDialog" name="SlacshEffectAttaqueDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>294</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Choisir effet</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Effets</string>
   </property>
   <widget class="QRadioButton" name="feuCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Feu (5 tours)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="poisonCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Poison (6 tours)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="coldCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cold (3 tours)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="stunCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stun (2 tours)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
